packages/desktopjs/tests/unit/container.spec.ts

```diff
@@ -85,6 +85,10 @@ export class TestContainer extends ContainerBase {
     public buildLayout(): Promise<PersistedWindowLayout> { return Promise.resolve(undefined); }
 
     public saveLayout(): Promise<PersistedWindowLayout> { return Promise.resolve(undefined); }
+
+    public setOptions(options: any) { }
+
+    public isAutoStartEnabledAtLogin(): Promise<boolean> { return Promise.resolve(undefined); }
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/brian.ingenito/desktopJS/pullRequest?prid=5916792)